apollo/src/resolvers/Status.js

```diff
@@ -0,0 +1,22 @@
+const labels = (parent, args, context) => {
+  const res = context.prisma.status({ id: parent.id }).labels();
+  return res;
```

Comment:
I'd add a line break between lines and 2 and 3 for consistency with the rest of the functions. 😄